src/spaced/spaced/game/damage.hpp

```diff
@@ -1,17 +1,27 @@
 #pragma once
-#include <bave/core/time.hpp>
-#include <glm/vec2.hpp>
+#include <bave/imgui/im_text.hpp>
+#include <bave/app.hpp>
```

Comment:
`bave/app.hpp` is not required. Instead of `glm::clamp` you can use `std::clamp` which is available in `<algorithm>`.